[2025-06-12T01:45:47.244+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dynamic_training_pipeline.split_data scheduled__2025-02-05T15:00:00+00:00 [queued]>
[2025-06-12T01:45:47.251+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dynamic_training_pipeline.split_data scheduled__2025-02-05T15:00:00+00:00 [queued]>
[2025-06-12T01:45:47.251+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-06-12T01:45:47.268+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): split_data> on 2025-02-05 15:00:00+00:00
[2025-06-12T01:45:47.277+0000] {standard_task_runner.py:60} INFO - Started process 724 to run task
[2025-06-12T01:45:47.280+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dynamic_training_pipeline', 'split_data', 'scheduled__2025-02-05T15:00:00+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/dag_dynamic.py', '--cfg-path', '/tmp/tmpwx988wyt']
[2025-06-12T01:45:47.282+0000] {standard_task_runner.py:88} INFO - Job 37: Subtask split_data
[2025-06-12T01:45:47.336+0000] {task_command.py:423} INFO - Running <TaskInstance: dynamic_training_pipeline.split_data scheduled__2025-02-05T15:00:00+00:00 [running]> on host caffbd336f08
[2025-06-12T01:45:47.385+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dynamic_training_pipeline' AIRFLOW_CTX_TASK_ID='split_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-05T15:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-05T15:00:00+00:00'
[2025-06-12T01:45:47.387+0000] {logging_mixin.py:188} INFO - Archivo merged.html no encontrado en: ./dags/2025-02-05/preprocessed/merged.html
[2025-06-12T01:45:47.387+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-12T01:45:47.393+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dynamic_training_pipeline, task_id=split_data, execution_date=20250205T150000, start_date=20250612T014547, end_date=20250612T014547
[2025-06-12T01:45:47.419+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-12T01:45:47.453+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
